source/requirements.rst

```diff
@@ -8,33 +8,17 @@ Software Requirements
 
 On the Web node serving OnDemand itself:
 
-- `epel-release`_
-- `centos-release-scl`_
+- `EPEL repository`_
+- `Software Collections repositories`_
 - `lsof`_
 - `sudo`_
-- `ondemand-release-web-latest-1-2.el7.noarch.rpm`_:
-    - cjose-0.6.1-1.el7.x86_64.rpm
-    - cjose-debuginfo-0.6.1-1.el7.x86_64.rpm
-    - cjose-devel-0.6.1-1.el7.x86_64.rpm
-    - httpd24-mod_auth_openidc-2.3.11-1.el7.x86_64.rpm
-    - httpd24-mod_auth_openidc-debuginfo-2.3.11-1.el7.x86_64.rpm
-    - ondemand-apache-{{ondemand_version}}-1.el7.x86_64.rpm
-    - ondemand-git-{{ondemand_version}}-1.el7.x86_64.rpm
-    - ondemand-nginx-1.14.0-2.p5.3.7.el7.x86_64.rpm
-    - ondemand-nginx-debuginfo-1.14.0-2.p5.3.7.el7.x86_64.rpm
-    - ondemand-nginx-filesystem-1.14.0-2.p5.3.7.el7.noarch.rpm
-    - ondemand-nodejs-{{ondemand_version}}-1.el7.x86_64.rpm
-    - ondemand-passenger-5.3.7-2.el7.x86_64.rpm
-    - ondemand-passenger-debuginfo-5.3.7-2.el7.x86_64.rpm
-    - ondemand-passenger-devel-5.3.7-2.el7.x86_64.rpm
-    - ondemand-ruby-{{ondemand_version}}-1.el7.x86_64.rpm
-    - ondemand-runtime-{{ondemand_version}}-1.el7.x86_64.rpm
-    - ondemand-scldevel-{{ondemand_version}}-1.el7.x86_64.rpm
-
-.. _centos-release-scl: https://www.softwarecollections.org/en/
-.. _epel-release: https://fedoraproject.org/wiki/EPEL
+- `OnDemand repository`_:
+    - ondemand-{{ondemand_version}}-1.el7.x86_64.rpm
```

Comment:
Should this now be `ondemand-{{ondemand_version}}-4.el7.x86_64.rpm`?

Reply:
No, that references the actual ondemand RPM. The one using `-4` is `ondemand-release-web`.